### DISCUSSION TITLE
Guided topic modelling returns an ValueError (inhomogenous shape)

### DISCUSSION THREAD

**Discussion by aneleh-cikab** [MaartenGr/BERTopic#1269] [Q&A]
I am trying to train a BERTopic model with a seed topic list. However, the model returns a Value Error:

> ValueError: setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimensions. The detected shape was (2,) + inhomogeneous part.

The same error happens when running the official tutorial example, so I assume there is an issue with changes in libraries. I am working with Python 3.10.5 and Numpy 1.24.3.

The example below:

```
from bertopic import BERTopic
from sklearn.datasets import fetch_20newsgroups

docs = fetch_20newsgroups(subset='all',  remove=('headers', 'footers', 'quotes'))["data"]

seed_topic_list = [["drug", "cancer", "drugs", "doctor"],
                   ["windows", "drive", "dos", "file"],
                   ["space", "launch", "orbit", "lunar"]]

topic_model = BERTopic(seed_topic_list=seed_topic_list, verbose=True, calculate_probabilities=False)
topics = topic_model.fit_transform(docs)
```

Thanks a lot for help!

<img width="688" alt="image" src="https://github.com/MaartenGr/BERTopic/assets/90176475/ce96e202-9e41-4c16-8d1a-d84674c5ad76">

**Comment by MaartenGr** [comment]
I did not get this issue when running it in a fresh Google Colab instance, so it might indeed have something to do with versions of dependencies. Could you share the versions of all packages in your environment? Also, it might be worthwhile to try out Python 3.9 instead since I remember there being issues with certain packages in 3.10.

**Reply by fali21DXC** [reply]
Hi MaartenGr upuntil today i was able to use topic_model.fit_transform(docs) and get the model to work but now i too am getting the above error...not sure why? any suggestions?i am using jupyter notebook on aws sagemaker....and it worked until this morning....but now it is not. the version on this is 3.10.13 for python....and this is my work related notebook fyi....

**Reply by fali21DXC** [reply]
Hi MaartenGr....once i changed to 3.9 it worked ...thanks
